src/docs/implementations/dynatrace.adoc

```diff
@@ -11,6 +11,52 @@ include::install.adoc[]
 
 == Configuring
 
+=== Quick setup
+
+For setting up new integrations with Dynatrace, it is recommended to use the latest version of the API (`api-version: v2`).
+
+Use the following code in your project to export Micrometer metrics to the specified Dynatrace endpoint. 
+https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/[The Dynatrace API token documentation] contains more information on how to create an API token.
+The metrics ingest (`metrics.ingest`) permission is required on the token in order to ingest metrics.
+More info on how to use the v2 API is provided <<bookmark-apiv2, below>>.
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String apiToken() {
+        return MY_TOKEN;
+    }
+
+    @Override
+    public String uri() {
+        return MY_DYNATRACE_URI;
+    }
+
+    @Override
+    @Nullable
+    public String get(String k) {
+        return null;
+    }
```

Comment:
Is it required to override this?

Reply:
I see. I think this is quite surprising for the very first basic example however. Maybe add a comment like "This method for retrieving arbitrary config items introduced by the interface is currently not used in DynatraceConfig.".